antidox/perspective.py

```diff
@@ -110,35 +111,31 @@ def contains_toxicity(perspective_response):
   return is_toxic
 
 
-def use_query(client, sql_query):
+def use_query(args.content, sql_query):
   """make big query api request"""
+  big_q = bigquery.Client.from_service_account_json(querkey.json)
   query_job = client.query(sql_query)
   rows = query_job.result()
   strlst = []
   for row in rows:
-    strlst.append(str(row.cleaned_content))
+    strlst.append(str(row.args.content))
```

Comment:
I think this may generate an error (row.args is probably undefined), instead you will want to do something more like `row[content]` where content is the variable name you are using in your function

Reply:
done